core/electron-manager/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@bentley/electron-manager",
```

Comment:
yes, but the Electron and Mobile cases are special in that you *always* need both the frontend and backend code at the same time in your package.json. Having them all in one package eliminates the possibility that you get them mismatched.

Reply:
I'm thinking about a case where the same app has both web and electron versions. I suppose in such case it would have separate packages for backend and frontend and the electron version would depend on both of them. I guess `electron-manager` would then be a dependency of only the electron app package and not the frontend, right? 

Just seems like adding a dependency to `electron-manager` from frontend-only package is an easy mistake to make. But maybe it's not a big deal since it only has peer dependencies and doesn't add a real dependency on `imodeljs-backend`.